tools/make/docs.mk

```diff
@@ -168,10 +168,6 @@ docs-release-gen:
 	$(eval DOC_VERSION := $(shell cat VERSION | cut -d "." -f 1,2))
 	@$(call log, "Added Release Doc: site/content/en/$(DOC_VERSION)")
 	cp -r site/content/en/latest/ site/content/en/$(DOC_VERSION)/
-	@echo "" >> site/hugo.toml
-	@echo '[[params.versions]]' >> site/hugo.toml
-	@echo '  version = "$(DOC_VERSION)"' >> site/hugo.toml
-	@echo '  url = "/$(DOC_VERSION)"' >> site/hugo.toml
```

Comment:
Remove this in order to:
 * Better control the eol date
 * Avoid generating multiple duplicated parms versions  when running `make docs-release` multiple times.
 
 Will add this in the release doc in a seperate PR.